### PR TITLE
Add converter for VotingRegressor

### PR DESCRIPTION
**Pull request by xadupre**

### CODE REVIEW

File: tests/test_sklearn_voting_regressor_converter.py [hunk 1]
```diff
@@ -0,0 +1,80 @@
+"""Tests GLMRegressor converter."""
```
Comment (prabhat00155) [inline tests/test_sklearn_voting_regressor_converter.py]: Typo here, VotingRegressor not GLMRegressor.

File: tests/test_sklearn_voting_regressor_converter.py [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+from sklearn.model_selection import train_test_split
+from sklearn.tree import DecisionTreeRegressor
+from skl2onnx import convert_sklearn
+from skl2onnx.common.data_types import (
+    FloatTensorType, Int64TensorType
+)
+from test_utils import dump_data_and_model
+
+
+def _fit_model(model, is_int=False):
```
Comment (prabhat00155) [inline tests/test_sklearn_voting_regressor_converter.py]: You may use https://github.com/onnx/sklearn-onnx/blob/d7a93a525c6ddfb8289a5c2d612755b8b9845885/tests/test_utils/tests_helper.py#L65, it does exactly the same thing.

Comment (xadupre) [reply tests/test_sklearn_voting_regressor_converter.py]: Done

File: tests/test_sklearn_voting_regressor_converter.py (hunk 1 above)
Comment (xadupre) [reply tests/test_sklearn_voting_regressor_converter.py]: Done

File: tests/test_sklearn_voting_regressor_converter.py [hunk 3]
```diff
@@ -0,0 +1,70 @@
+"""Tests VotingRegressor converter."""
+
+import unittest
+import numpy
+from sklearn import datasets
```
Comment (prabhat00155) [inline tests/test_sklearn_voting_regressor_converter.py]: This is no longer used, can you remove it? I am surprised flake8 check didn't fail.

File: tests/test_sklearn_voting_regressor_converter.py [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+from sklearn.linear_model import LinearRegression
+try:
+    from sklearn.ensemble import VotingRegressor
+except ImportError:
+    # New in 0.21
+    VotingRegressor = None
+from sklearn.model_selection import train_test_split
```
Comment (prabhat00155) [inline tests/test_sklearn_voting_regressor_converter.py]: Unused import

File: tests/test_sklearn_voting_regressor_converter.py (hunk 3 above)
Comment (xadupre) [reply tests/test_sklearn_voting_regressor_converter.py]: Done

**Pull request merged**